Izrada modal za dodavanje recepta, spremanje u local s. izrada dugmeta na receptu i dvije liste za bookmark, koje ce spremat oznacavat i brisat ili dodavat recept u local storage

AhmedMehmedovic opened the pull request:
* Commit od 30.09 petak

* Commit za REAC-1 I REAC-4 spremanje u localstorage

* REAC-1 Izmjene da sajt funkcionise, izmjena

* REAC-1 Popravak omogućavanje dodavanja više

* Ispravka (amel) dodavanje button u bookmark da se

* Nastavak

* REAC-1 zavrsene nadogradnje za dugme bookmark u

Zavrsena 2 zadatka od 6 planiranih. Lose planiranje jire

maniaba committed:
Merge pull request #1

* Commit od 30.09 petak

* Commit za REAC-1 I REAC-4 spremanje u localstorage

* REAC-1 Izmjene da sajt funkcionise, izmjena

* REAC-1 Popravak omogućavanje dodavanja više

* Ispravka (amel) dodavanje button u bookmark da se

* Nastavak

* REAC-1 zavrsene nadogradnje za dugme bookmark u

maniaba commented on src/js/helpers.js:
```diff
@@ -8,7 +8,29 @@ const timeout = function (s) {
+export const AJAX = async function (url, uploadData = undefined) {
+  try {
+    const fetchPro = uploadData
+      ? fetch(url, {
```
Zašto svaki puta proslijediš cijeli url, a ovdje možeš koristit const varijablu

maniaba commented on src/js/controller.js:
```diff
@@ -99,31 +102,97 @@ const controlServings = function (newServings) {
+    // window.history.back() klikom u browseru za nazad automatski idemo nazad
+
+    /// close form window
+    setTimeout(function () {
```
setTimeout nemoj koristit

maniaba commented on src/js/view/addRecipeView.js:
```diff
@@ -2,9 +2,59 @@ import View from './view';
+      handler(data);
+    });
+  }
   _generateMarkup() {}
```
Zašto imas praznu metodu??

maniaba commented on src/js/controller.js:
```diff
@@ -99,31 +102,97 @@ const controlServings = function (newServings) {
+
+    //succes message
+
+    console.log(addRecipeView.renderMessage);
```
console.log u master??

AhmedMehmedovic committed:
Ispravke na pregledu :
1. Zasto prosljedjujem url, a ne const varijablu ("API_URL") proslijedim uz url i API_URL ali ne funkcionise, ne zna zasto.
2. SetTimeout necu vise koristiti
3. Uklonjena prazna metoda
4. Pregledani svi fajlovi ukolnjeni nepotrebni consol.log osim u catch gdje ispisuju gresku

AhmedMehmedovic committed:
Izmijenjen consol. log u consol.error u model.js linija 44

AhmedMehmedovic committed:
Ispravljene greške sa review
1. zamjena API_URL u helpers. js(${API_URL+url}),
2. uklanjanje console logova
3. pregled koristenja setTimeout, zabrana daljeg koristenja

AhmedMehmedovic committed:
Zadnji commit
Vscode zeznu ponovo moram comit radit zato sto sam prije zavrsetka unosa poruke izbrisao preostale console logove
